db/migrations/1_add_forms_table.rb

```diff
@@ -0,0 +1,11 @@
+Sequel.migration do
+  change do
+    create_table :forms do
+      primary_key :id, type: :Bignum
```

Comment:
@danielburnley is this supposed to be a string? If I'm reading the ruby code correctly, the filename is used as the key?